### DISCUSSION TITLE
Player Can't Enter Volatus other than as Driver

### DISCUSSION THREAD

**Discussion by AliciaWindsor** [scripthookvdotnet/scripthookvdotnet#1480] [General]
This is an incredibly strange issue. Probably doesn't have a solution. To preface, I have used the Enter Vehicle task, both the native function and the SHVND 3 one, dozens of times, to have the player or ped assigned to a car or plane without any issues.  For example:
Game.Player.Character.Task.EnterVehicle(Vehicle1, VehicleSeat.RightFront);
Today however, I spent 45 minutes trying to understand why this didn't work with a helicopter. Native function didn't work, SHVDN didn't work. Player always spawned as the driver/pilot. If I used the warp function instead, then the player correctly entered the vehicle in the correct seat. Eventually I realized this issue was with a specific model - the volatus. I changed the model to Maverick and it fixed it immediately. This is frustrating because I wanted the volatus and I want the player to walk to the helicopter, not warp into it.

**Comment by Tivertoni** [comment]
This seems to be a lazy implemented feature by R*, I created an issue for this at #1677.
